byzcoin/heartbeat.go

```diff
@@ -17,17 +17,18 @@ type heartbeat struct {
 	getTimeChan chan chan time.Time
 	timeout     time.Duration
 	timeoutChan chan string
+	updateTO    chan time.Duration
```

Comment:
s/updateTO/updateTimeoutChan/ please. It's too hard to know if to is like to/From or to is timeout.